src/components/common/VerifiedIcon.tsx

```diff
@@ -0,0 +1,34 @@
+import { Circle, Container, Icon, SquareProps, Tooltip } from "@chakra-ui/react"
+import { Check, CircleWavy } from "phosphor-react"
+
+type Props = {
+  iconSize: number
+} & SquareProps
+
+const VerifiedIcon = ({ iconSize, ...chakraProps }: Props): JSX.Element => (
```

Comment:
If you make some changes inside GuildCard (see my comment there), you can remove `...chakraProps` (`SquareProps`) here